wiki/Beatmap_Editor_Guides/Compressing_Videos/en.md

```diff
@@ -19,40 +19,40 @@ Removing audio
 
    a) Press either (no is better unless the program crashes).
 
-![](CV_01.png "CV_01.png")
+![Tutorial Image 1](CV_01.png "Tutorial Image 1")
 
 4) Select Audio -> Main Track
 
-![](CV_02.png "CV_02.png")
+![Tutorial Image 2](CV_02.png "Tutorial Image 2")
 
 5) Select Audio Source -> None
 
-![](CV_03.png "CV_03.png")
+![Tutorial Image 3](CV_03.png "Tutorial Image 3")
 
 Encoding Video
 --------------
 
 6) Make sure that Video and Audio are on "Copy".
```

Comment:
L35: Revise. Suggested ``Make sure that both the video and audio are on "Copy"``